### PR TITLE
chore: apply review suggestions to smart-contract

### LINKED ISSUE #16: [TODO] More Beta Prep
- [x] Collapse JSON output in `Console`
- [x] Congratulations message (indicator that Camper "won");
- [x] Markdown in `Tests`
- [x] Spacing in `Tests`
- [x] Remove list-styling in `Tests`
- [x] Loading screen needs work (extension)
- [x] Simple Browser opens before server is ready
- [x] In-line codeblock overlaps - add better padding, etc.

**Build a Smart Contract in Rust**

- [x] Mention specific file/directory Campers need to work in
- [x] Add `wasm-pack` to `Dockerfile`
- [x] Add `Account`, `Context` structs to **User Stories**
- [x] Mention how to use `JsValue` - how to wrap a Struct in a `JsValue`
- [ ] ~Add `_blank` to external links~ _not possible due to Simple Browser sandbox_
- [x] Remove first test `lib::`
- [x] Mention importing needed imports
- [ ] "If you want to see your contract in action..."
  - [ ] Deploy contract
  - [ ] Run nodes
  - [ ] Test contract

### LINKED ISSUE #19: [REVIEW] Build a Smart Contract in Rust
The `build-a-smart-contract-in-rust` project could do with a section/bonus/requirement for Campers to:

-  "If you want to see your contract in action..."
  - Deploy contract
  - Run nodes
  - Test contract

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #19

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
This should be ready. Be weary of that last commit - I am not sure if that breaks anything else, but it is necessary as it was/is a bug.

File: .freeCodeCamp/tooling/test-utils.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ async function getTerminalOutput() {
  * @returns {{stdout, stderr}}
  */
 async function getCommandOutput(command, path = '') {
+  console.log('TEST: !!!!! ', ROOT, path);
```
Comment (moT01) [inline .freeCodeCamp/tooling/test-utils.js]:
```suggestion
```

File: .freeCodeCamp/tooling/locales/english/build-a-smart-contract-in-rust.md [hunk 2]
```diff
@@ -6,9 +6,9 @@
 
 You will build a Smart Contract using Rust.
```
Comment (moT01) [inline .freeCodeCamp/tooling/locales/english/build-a-smart-contract-in-rust.md]:
```suggestion
For this project, you need to create and deploy Smart Contract using Rust that keeps track of how many users have clicked a button on your web page.
```

This isn't a specific suggestion - but a suggestion to make it more specific. Something to let users know a little more about what they will build. I added `For this project` at the start - one of my projects has that, two of them don't. I was thinking we could maybe add it to all of them.

**Review by moT01 — COMMENTED** [review]
I didn't go over this too hard - but here's a few things I noticed...

I ran the tests without adding any code - seemed like they were hanging, but I think they were just taking a while to run/compile?/install?/build?/whatever they are doing. A second run with no additions, ran very fast. A third run after adding the solution took a while again. If this is expected, I would add a note that they will take a while to run. Maybe "Note: The tests need to (do whatever they need to do), so they may take a while the first time you add or change code."

I couldn’t deploy the contract - I probably wasn’t doing it right. Didn’t look into too hard - we can go over this together again and you can show me.

I would be fine merging this whenever you want @ShaunSHamilton and we can iterate on it.

**Comment by moT01** [comment]
> Be weary of that last commit - I am not sure if that breaks anything else, but it is necessary as it was/is a bug.

No worries on this - it may break many of my tests, but it should be an easy fix.

**Pull request merged**